backend/tests/unit/simulation/alarms.txt

```diff
@@ -0,0 +1,34 @@
+Scenario: alarm services behaving properly
+Given: a pc_ac alarm service object 
+When: parameter mode is not equal to pc_ac
+Then: alarms are not updated for parameters
+
+Scenario: alarm services behaving properly
+Given: a pc_ac alarm service object 
+When: parameter ventilating mode is false
+Then: alarms are not updated for parameters
+
+Scenario: alarm services behaving properly
+Given: a pc_ac alarm service object 
+When: sensor measurments spo2 is less than lower alarm_limit for spo2
+Then: alarm for spo2 too low is triggered
```

Comment:
The alarm should only be triggered if there isn't already an active alarm on SpO2 too low, otherwise, the previous alarm for that same condition continues to remain active. This comment also applies to the FiO2 alarms.

Reply:
Perhaps the different conditions of whether an alarm is already active should be expressed as part of the "Given" statement.